infra/base-images/base-runner/gocoverage/llvm-cov-rel/llvm-profraw-relative.go

```diff
@@ -0,0 +1,180 @@
+package main
```

Comment:
In the future I would prefer code for OSS-Fuzz be written in python.
I still need to figure out how to remove the go toolchain from the runner image because we didn't stick to this policy. All of the other code in OSS-Fuzz is python or bash (also an unfortunate choice), lets try to do python only from now on.

Reply:
So, do you want me to rewrite this in python ?

Reply:
Not necessary for this PR, but in future, I'd prefer it if we had as little go code as possible yes.

Reply:
This PR LGTM, but if it won't take to long, could you please convert this script to Python?
Thanks!